flow/layers/layer_tree.cc

```diff
@@ -159,4 +150,32 @@ sk_sp<SkPicture> LayerTree::Flatten(const SkRect& bounds) {
   return recorder.finishRecordingAsPicture();
 }
 
+void LayerTree::UpdateScene(SceneUpdateContext& context, scenic::ContainerNode& container) {
+#if defined(OS_FUCHSIA)
+  TRACE_EVENT0("flutter", "LayerTree::UpdateScene");
+
+  // Ensure the context is aware of the view metrics.
+  context.set_frame_dimensions(frame_size_, frame_depth_, frame_pixel_ratio_);
+
+  const auto& metrics = context.metrics();
+  SceneUpdateContext::Transform transform(context,                  // context
+                                          1.0f / metrics->scale_x,  // X
+                                          1.0f / metrics->scale_y,  // Y
+                                          1.0f / metrics->scale_z   // Z
```

Comment:
Are these all guaranteed to be non-zero?

Reply:
Good catch; they don't seem to be.  I'll add some defensive DCHECKS.

Reply:
Nope, nothing is checking them right now, good catch.  I'll add some DCHECKs